packages/mc-scripts/src/bin/mc-scripts.js

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+const mri = require('mri');
+const prompts = require('prompts');
+const minimatch = require('minimatch');
+const dotenv = require('dotenv');
+const dotenvExpand = require('dotenv-expand');
+const { getPackages } = require('@manypkg/get-packages');
+const spawn = require('react-dev-utils/crossSpawn');
+
+const flags = mri(process.argv.slice(2), { alias: { help: ['h'] } });
+const commands = flags._;
+
+if (commands.length === 0 || (flags.help && commands.length === 0)) {
+  console.log(`
+Usage: mc-scripts [global-options] [command] [options]
+
+Global options:
+
+  --env <path>              (optional) Parses the file path as a dotenv file and adds the variables to the environment. Multiple flags are allowed.
+  --env-type <environment>  (optional) Supports cascading env variables from ".env", ".env.local", ".env.<environment>", ".env.<environment>.local" files.
```

Comment:
The below is more of a short hand of the before right. I could achieve `--env-type` with `--env` more inconvienetly?

Reply:
This is the equivalent option of `-c` from `dotenv-cli`, I just named this differently. https://www.npmjs.com/package/dotenv-cli#cascading-env-variables

However, if you pass `--env-type`, the `--env` option(s) is ignored (see implementation of `dotenv-cli`).